src/dotnet/commands/dotnet-publish3/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Tools.MSBuild;
+
+namespace Microsoft.DotNet.Tools.Publish3
+{
+    public class Publish3Command
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false);
+            app.Name = "dotnet publish3";
+            app.FullName = ".NET Publisher";
+            app.Description = "Publisher for the .NET Platform";
+            app.AllowArgumentSeparator = true;
+            app.HelpOption("-h|--help");
+
+            CommandArgument projectArgument = app.Argument("<PROJECT>",
+                "The MSBuild project file to publish. If a project file is not specified," +
+                " MSBuild searches the current working directory for a file that has a file extension that ends in `proj` and uses that file.");
+
+            CommandOption frameworkOption = app.Option("-f|--framework <FRAMEWORK>", "Target framework to publish for", CommandOptionType.SingleValue);
+            CommandOption runtimeOption = app.Option("-r|--runtime <RUNTIME_IDENTIFIER>", "Target runtime to publish for", CommandOptionType.SingleValue);
+            CommandOption outputOption = app.Option("-o|--output <OUTPUT_DIR>", "Path in which to publish the app", CommandOptionType.SingleValue);
```

Comment:
I dislike that requirement when the project targets a single TFM.

We don't have an "outer build" for "Publish", and I'm not sure we need one....  @nguerrera ??

My thinking is that "Publish" is an "inner build" only operation.  If you try to call it against a project that has multiple TFMs, you should get an error.